src/plugins/intel_gpu/include/intel_gpu/op/rms.hpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/op.hpp"
+
+namespace ov {
+namespace intel_gpu {
+namespace op {
+
+class RMS : public ov::op::Op {
```

Comment:
Please add a short operation description here

Reply:
Added description for RMS operator